docs/index.html

```diff
@@ -8,7 +8,7 @@
             <title>consumerfinance.gov lighthouse reports</title>
         
         
-            <link rel="stylesheet" href="/cfgov-lighthouse/static/css/main.css">
+            <link rel="stylesheet" href="/static/css/main.css">
```

Comment:
These path changes seem unrelated to the rest of the PR, is this intended?

Reply:
Yeah it's a difference in our local setup versus not and were generated with `yarn && yarn start`, but @chosak tells me committing them doesn't matter.